paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/FlinkCatalogTest.java

```diff
@@ -428,12 +428,11 @@ public void testCreateDb_DatabaseAlreadyExistException() throws Exception {
     }
 
     @Test
-    public void testCreateDb_DatabaseWithPropertiesException() {
+    public void testCreateDb_DatabaseWithProperties() throws Exception {
         CatalogDatabaseImpl database =
                 new CatalogDatabaseImpl(Collections.singletonMap("haa", "ccc"), null);
-        assertThatThrownBy(() -> catalog.createDatabase(path1.getDatabaseName(), database, false))
-                .isInstanceOf(UnsupportedOperationException.class)
-                .hasMessage("Create database with properties is unsupported.");
+        catalog.createDatabase(path1.getDatabaseName(), database, false);
+        assertThat(catalog.databaseExists(path1.getDatabaseName())).isTrue();
```

Comment:
Can we add validation for props existence

Reply:
Thanks @Zouxxyy , done